script/utils/update-content-versions.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
+import path from 'path';
+import { makeUnifiedBookLoader } from '../../src/app/content/utils';
+import { ARCHIVE_URL, REACT_APP_ARCHIVE_URL, REACT_APP_OS_WEB_API_URL } from '../../src/config';
+import books from '../../src/config.books';
+import createArchiveLoader from '../../src/gateways/createArchiveLoader';
+import createOSWebLoader from '../../src/gateways/createOSWebLoader';
+
+export interface SimpleBook {
+  bookId: string;
+  versionNumber: string;
+}
+
+const booksPath = path.resolve(__dirname, '../../src/config.books.json');
+
+const bookLoader = (newArchiveUrl: string) => makeUnifiedBookLoader(
+  createArchiveLoader(newArchiveUrl, {
+    archivePrefix: ARCHIVE_URL,
+  }),
+  createOSWebLoader(`${ARCHIVE_URL}${REACT_APP_OS_WEB_API_URL}`)
+);
+
+async function processBookVersionUpdate(book: SimpleBook, newArchive: string) {
+  const {bookId, versionNumber} = book;
+  const { defaultVersion } = books[bookId] || {};
+  const newVersion = versionNumber.toString();
+
+  if (defaultVersion === newVersion) {
+    // tslint:disable-next-line: no-console
+    console.log(`${bookId} already at desired version.`);
+    return;
+  }
+
+  const { title, version } = await bookLoader(newArchive)(bookId, newVersion)
+    .catch((error) => {
+      // tslint:disable-next-line: no-console max-line-length
+      console.log(`error while loading book ${bookId} with version ${newVersion} using pipeline ${newArchive || REACT_APP_ARCHIVE_URL}`);
+      throw error;
+    });
```

Comment:
actually do you even need to load this? isn't the loaded version the same as the input version? and the title is just for debug output.

we don't necessarily have to validate that the book exists, because the PR checks will get it. maybe you can actually remove this?